src/applications/personalization/profile/components/ProfileInformationEditView.jsx

```diff
@@ -334,7 +334,7 @@ export class ProfileInformationEditView extends Component {
                     data-testid="save-edit-button"
                     isLoading={isLoading}
                     loadingText="Saving changes"
-                    className="vads-u-margin-top--0"
+                    className="vads-u-margin-top--0 medium-screen:vads-u-width--auto"
```

Comment:
Was there a styling defect that yall saw that required this class to be added?

Reply:
Yes, as currently implemented they're taking the full width of the container at all screen sizes causing the display to be misaligned with design.

Reply:
Okay thanks! Verified that it still looks the same on the Profile application with this class added, so approved.

Reply:
Sure thing! I actually made another change because our design team wanted the buttons slightly bigger. Below is what we now have and desire on our side. I can wait for your double check to merge if you'd like!

![image](https://github.com/department-of-veterans-affairs/vets-website/assets/29178824/0feb0213-3953-4eb8-83d5-c7cc5e1f4cb1)